src/components/Powertools/index.tsx

```diff
@@ -1,34 +1,120 @@
-import { graphql } from 'gatsby'
-import React from 'react'
+import { graphql, useStaticQuery } from 'gatsby'
+import Img from 'gatsby-image'
+import React, { FC, ReactElement } from 'react'
 import ReactMarkdown from 'react-commonmark'
-import { Col, Container, Row } from 'reactstrap'
+import { FaChrome, FaFirefox, FaSafari } from 'react-icons/fa'
+import { Button, Col, Container, Row } from 'reactstrap'
 
-import { ContactForm } from '../Contact'
+import { mapToComponent } from '../../utils/helpers'
 import { TitleBanner } from '../TitleBanner'
-import CommingSoon from './commingSoon.md'
-import image from './extensionScreenShot.png'
-
-export const PowertoolsComponent = () => (
-  <article>
-    <TitleBanner
-      title="Powertools Browser Erweiterung"
-      subtitle="Versenden Sie Ihre Rechnungen direkt aus Lexoffice per Brief an Ihre Kunden."
-    />
-    <Container>
-      <Row className="mt-5 mb-3">
-        <Col className="pb-4">
-          <img
-            className={'img-fluid pb-5'}
-            alt={`Extension Screenshot`}
-            src={image}
-          />
-        </Col>
-      </Row>
-      <Row>
-        <Col xs={12}>
-          <ReactMarkdown source={CommingSoon} />
-        </Col>
-      </Row>
-    </Container>
-  </article>
+import PowertoolsIcon from './assets/extensionLogoTransparent.svg'
+import Description from './description.md'
+
+const extensionChromeStoreLink =
+  'https://chrome.google.com/webstore/detail/hardfork-powertools/ocfgkhnplliadmaifkhhjncimaobcchh'
+
+const downloadItems: DownloadItemProps[] = [
+  {
+    link: extensionChromeStoreLink,
+    available: true,
+    browser: 'Chrome',
+    icon: <FaChrome className="mr-2 mr-md-3" size="2em" />,
+  },
+  {
+    link: '',
+    available: false,
+    browser: 'Firefox',
+    icon: <FaFirefox className="mr-2 mr-md-3" size="2em" />,
+  },
+  {
+    link: '',
+    available: false,
+    browser: 'Safari',
+    icon: <FaSafari className="mr-2 mr-md-3" size="2em" />,
+  },
+]
+
+const extensionScreenShotQuery = graphql`
+  query extensionShot {
+    extensionShot: file(relativePath: { eq: "extensionScreenShot.png" }) {
+      childImageSharp {
+        fluid(maxWidth: 1280, quality: 100) {
+          ...GatsbyImageSharpFluid
+        }
+      }
+    }
+  }
+`
+
+export const PowertoolsComponent = () => {
+  const data = useStaticQuery(extensionScreenShotQuery)
+
+  return (
+    <article>
+      <TitleBanner
+        title="Browser Erweiterung"
+        subtitle="Versenden Sie Ihre Rechnungen direkt aus Lexoffice per Brief an Ihre Kunden."
+      />
+      <Container>
+        <Row className="mt-5 mb-3 ">
+          <Col className="pb-4 justify-content-center">
+            <PowertoolsIcon className="img-fluid" />
+          </Col>
+        </Row>
+        <ReactMarkdown source={Description} />
+        <Row className="mt-5 mb-3">
+          <Col xs={12}>
+            <Downloads />
+            <h2>Screenshots</h2>
+
+            <Img
```

Comment:
This image should have have a style with `height: auto` in order to prevent the croppig on the edges. So far I could not figure it out how to set is...